packages/astro/src/assets/utils/resolveImports.ts

```diff
@@ -16,7 +16,7 @@ export function imageSrcToImportId(imageSrc: string, filePath?: string): string
 	imageSrc = removeBase(imageSrc, IMAGE_IMPORT_PREFIX);
 
 	// We only care about local imports
-	if (isRemotePath(imageSrc) || imageSrc.startsWith('/')) {
+	if (isRemotePath(imageSrc)) {
```

Comment:
note: `isRemote()` incorrectly assumes remote URLs start with a protocol, so the code now won't catch `//foo.com/bar.jpg` anymore, which it still should?